Dshop/apps/products_catalogue/migrations/0001_initial.py

```diff
@@ -1,46 +1,122 @@
-# Generated by Django 4.2.3 on 2023-08-06 17:58
```

Comment:
Please revert all the migration changes and add yours as new ones. Removing those migrations will break database for everybody in the team. 
Migrations are like version control (git) for DB changes. If you delete git commits, everyone else will have issues with the repo. Same thing here.